go.mod

```diff
@@ -50,19 +51,20 @@ require (
 	k8s.io/kube-openapi v0.0.0-20210527164424-3c818078ee3d
 	kubesphere.io/api v0.0.0-20210511124541-08f2d682bd07
 	sigs.k8s.io/controller-runtime v0.6.3
-	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
 replace (
 	github.com/fatih/structs => github.com/fatih/structs v1.1.0
 	github.com/go-redis/redis => github.com/go-redis/redis v6.15.2+incompatible
 	github.com/golang/example => github.com/golang/example v0.0.0-20170904185048-46695d81d1fa
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.0
+	github.com/jenkins-zh/jenkins-cli => /root/go/src/kubesphere.io/devops/pkg/client/devops/jenkins-cli
```

Comment:
The directory `/root/go/src/kubesphere.io/devops/pkg/client/devops/jenkins-cli` might not exist in other contributors' environments.

Reply:
Thank you for your comments. In my new commits, I introduced `jenkins-cli` as a submodule. There will be no absolute directory path in `go.mod`.